framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageTransportBindingElement.cs

```diff
@@ -0,0 +1,303 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.ServiceModel.Channels;
+using System.Xml;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+    /// <summary>
+    /// AzureQueueStorage Binding Element.  
+    /// Used to configure and construct AzureQueueStorage ChannelFactories and ChannelListeners.
+    /// </summary>
+    public class AzureQueueStorageTransportBindingElement
+        : TransportBindingElement // to signal that we're a transport
+    {
+        bool multicast;
+        static XmlDocument xmlDocument;
```

Comment:
multicast goes away as it's a udp concept

Reply:
Yes. removed